Kubernetes/HOWTO-on-prem.md

```diff
@@ -86,37 +118,53 @@ Run the script `start-kubelet.sh`. You may need to run it with `sudo`. Then, in
 
     ~/kube $ sudo cp ~/.kube/config kubelet/
 
-**TODO**: Why is this necessary? Shouldn't this be done automatically by one of the containers?
+**TODO**: Why is this necessary? Shouldn't this be done automatically by one of the containers? **What does this do?**

```

Comment:
One of the containers (IIRC the API server) mounts the Kubernetes configuration directory here, and then looks for the `config` file to set up various certificate stuff.